packages/taro-router/rollup.config-1745118866046.mjs

```
@@ -0,0 +1,51 @@
+import commonjs from '@rollup/plugin-commonjs';
```

ESLint

[error] Extra semicolon.

(semi)